infrastructure/k8s/services/auth.yaml

```diff
@@ -0,0 +1,42 @@
+project_name: "taskcluster-auth"
+secrets:
+  aws_access_key_id: ".Values.awsAccessKeyId"
+  aws_region: ".Values.awsRegion"
+  aws_secret_access_key: ".Values.awsSecretAccessKey"
+  azure_account: ".Values.tfAzureAccount"
+  azure_accounts: "" # FIXME this was "${jsonencode(map( "${azurerm_storage_account.base.name}", "${azurerm_storage_account.base.primary_access_key}",))}"
```

Comment:
As a first pass I'd put the json for `{account_name: access_key}` as a single string in sops.